package.json

```
@@ -10,7 +10,8 @@
     "lint": "eslint src buildtools examples",
     "checks": "npm run lint && npm run typecheck",
     "doc": "typedoc --name ol-cesium --excludeExternals --out apidoc --entryPoints src/olcs.ts --tsconfig ./tsconfig.json",
-    "start": "parcel serve --no-autoinstall --target examples",
+    "prestart": "rm -rf dist; mkdir -p dist/examples/node_modules/ol; mkdir -p dist/examples/node_modules/cesium/Build/; cp examples/inject_ol_cesium.js dist/examples/; cp node_modules/ol/ol.css dist/examples/node_modules/ol/; cp -R node_modules/cesium/Build/CesiumUnminified node_modules/cesium/Build/Cesium dist/examples/node_modules/cesium/Build/; cp -R examples/data dist/examples/",
```

Instead of duplicating you can do like this:
```suggestion
    "copy-static-dist": "mkdir -p dist/examples/node_modules/ol; mkdir -p dist/examples/node_modules/cesium/Build/; cp examples/inject_ol_cesium.js dist/examples/; cp node_modules/ol/ol.css dist/examples/node_modules/ol/; cp -R node_modules/cesium/Build/CesiumUnminified node_modules/cesium/Build/Cesium dist/examples/node_modules/cesium/Build/; cp -R examples/data dist/examples/"
    "prestart": "rm -rf dist; npm run copy-static-dist",
    "build-examples": "rm -rf dist; parcel build && npm run copy-static-dist",
```

---

added the `dist-dir` parameter to build-examples, it was taking it from the target